### DISCUSSION TITLE
Font Sizing Issues Introduced With/After GitHub iOS App v1.102.0's Release

### DISCUSSION THREAD

**Discussion by RandomDSdevel** [community/community#48916] [Mobile]
### Select Topic Area

Product Feedback

### Body

&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;The release notes for v1.102.0 of the GitHub iOS app include the following:  

> ⋮
> 
> Bug Fixes
>  - Code within commit and pull request diffs scales with Dynamic Type sizes.  
> ⋮

This is good/nice in theory.  I've sometimes wanted to make code listings a bit bigger or zoom in on them without having to resort to the system-wide iOS accessibility zoom feature.  Now code's _too_ big, though.  It appears to be using the Dynamic Type font size meant for headings, except without bolding. or a size just below that?  The previous size only a _little_ too small, by like maybe a point or two, and then only sometimes.  (It depends on whether there's a sequence of characters whose boundaries or shapes are harder to tell apart in view.)  
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;v1.102.0 also changed the sizes for fonts used in the 'Notifications' tab.  Now:  

 - Entry titles use the Dynamic Type heading font size and, 
 - Like code in commit and PR diffs, entry descriptions' preview text here also uses that same font size, except without bolding.  

I was fine with how fonts in the Notifications' tab were sized before v1.102.0.  
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Could users maybe have the ability to designate Dynamic Type font sizes and styles used for various different kinds of in-app text independently in in-app settings?  The pre- and post-v1.102.0 sets of behavior could then just be batch presets for these settings.  (For some additional context, I use iOS's system text font size accessibility settings, but only just _barely_, to bump body text up in font size from, say, 10 or 12 pts up 12 or 14 pts.  
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;(Perhaps iOS Settings per-app accessibility settings have me covered here, but I haven't fiddled with them just yet.  I'll go take a look at those.)

**Comment by RandomDSdevel** [comment]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;No, giving GitHub's iOS app different per-app font accessibility settings makes other text in the app just a tad too small.

**Comment by RandomDSdevel** [comment]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;It just occurred to me to check in the iOS app's diff view settings for any options for changing the font size of text there, and I found the 'Override System Font Size' setting.  Adjusting it resolves this issue for the diff view.  Notification view entry titles could still use an option to go back to not using heading style, though, as they're still too big.

**Comment by RandomDSdevel** [comment]
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Oh, and the font size used for commit description text is still too big now.

**Reply by markaschneider** [reply]
Agreed, the font size is too big in various places.

Why can't the app let us set all of the font sizes?
